docker_entrypoint.sh

```diff
@@ -40,12 +48,13 @@ do
     sleep 1
 done
 cp /usr/local/libexec/c-lightning/plugins/c-lightning-REST/certs/access.macaroon /root/.lightning/public/access.macaroon
-cat /root/.lightning/public/access.macaroon | base64  > /root/.lightning/start9/access.macaroon.base64
-cat /root/.lightning/public/access.macaroon | base64 | xxd  > /root/.lightning/start9/access.macaroon.hex
+cat /root/.lightning/public/access.macaroon | basenc --base64url -w0  > /root/.lightning/start9/access.macaroon.base64
```

Comment:
:speech_balloon: Thanks for fixing, in a side conversation learned.
> they were producing extra output
the base64 you had put whitespace, and also was not url safe
and the hex had a whole bunch of formatting to see the bytes and their ascii encodings